<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>501</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>878</width>
    <height>501</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>878</width>
    <height>501</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="search_ui">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>711</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="searchLine">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>请输入查找内容</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>卡号</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>种类</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>姓名</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>单位</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>挂失</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>有效性</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>查找</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>61</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>卡号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>姓名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>种类</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>单位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>挂失</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>有效性</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>601</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>601</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>240</y>
        <width>601</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>290</y>
        <width>601</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>340</y>
        <width>601</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>400</y>
        <width>601</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tongji_ui">
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>统计</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="luru_ui">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>241</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>种类</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="kinds">
           <item>
            <property name="text">
             <string>教工卡</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>学生卡</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>临时卡</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>其他卡</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>姓名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>单位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="company"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>360</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>录入</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="zhuce_ui">
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>请输入用户名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>150</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>请输入密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="userName">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="password1">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="password2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>190</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>确认密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>250</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>250</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionsearching"/>
    <addaction name="actiontongji"/>
    <addaction name="actionluru"/>
    <addaction name="actionzhuce"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="actionguashi"/>
    <addaction name="actionjiegua"/>
    <addaction name="actionzhuxiao"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>消费</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>使用</string>
     </property>
     <addaction name="actionxiaoyuan"/>
     <addaction name="actionxiaowai"/>
    </widget>
    <addaction name="actionchongzhi"/>
    <addaction name="menu_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsearching">
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="actiontongji">
   <property name="text">
    <string>统计</string>
   </property>
  </action>
  <action name="actionluru">
   <property name="text">
    <string>录入</string>
   </property>
  </action>
  <action name="actionzhuce">
   <property name="text">
    <string>注册</string>
   </property>
  </action>
  <action name="actionguashi">
   <property name="text">
    <string>挂失</string>
   </property>
  </action>
  <action name="actionjiegua">
   <property name="text">
    <string>解挂</string>
   </property>
  </action>
  <action name="actionzhuxiao">
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="actionchongzhi">
   <property name="text">
    <string>充值</string>
   </property>
  </action>
  <action name="actionxiaoyuan">
   <property name="text">
    <string>校园一卡通</string>
   </property>
  </action>
  <action name="actionxiaowai">
   <property name="text">
    <string>公交一卡通</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
